src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -39,6 +42,19 @@
     public static final String VALID_NOTE_AMY = "Amy";
     public static final String VALID_NOTE_BOB = "Bob";
 
+    public static final String VALID_NAME_TASK1 = "Task 1";
+    public static final String VALID_NAME_TASK2 = "Task 2";
+    public static final String VALID_DATE_TASK1 = "16-05-2022";
+    public static final String VALID_DATE_TASK2 = "04-08-2022";
+    public static final String VALID_STARTTIME_TASK1 = "09:00";
+    public static final String VALID_STARTTIME_TASK2 = "10:00";
+    public static final String VALID_ENDTIME_TASK1 = "12:00";
+    public static final String VALID_ENDTIME_TASK2 = "13:00";
+    public static final String VALID_TAG_EVENT = "Event";
+    public static final String VALID_TAG_MEET = "Meet";
+
```

Comment:
I think it will be a good idea to add some invalid values to support future test cases for task commands